greeting/src/htmlgreeter.ts

```diff
@@ -0,0 +1,11 @@
+import { Greeter } from "../src/greeter";
+
+export class HtmlGreeter extends Greeter {
+  constructor(greeting: string, private tagName: string = "h1") {
```

Comment:
You have a constructor here, but your are missing the "this.tagName = tagName" inside of the constructor. 

```
export class HtmlGreeter extends Greeting {
  tagName: string;

  constructor(greeting: string, tagName: string) {
    super(greeting);
    this.tagName = tagName;
  }

  greet(name: string): string {
    return `<${this.tagName}>${this.greeting}, ${name}!</${this.tagName}>`;
  }
}
```